app-web/src/components/Home/Masthead.js

```diff
@@ -60,6 +66,15 @@ const IconDiv = styled.div`
 `;
 
 export const Masthead = ({ query, resultCount, searchSourcesLoading }) => {
+  const { authenticated } = useAuthenticated();
+  let visible = !(query === '');
```

Comment:
I feel this could be simplified to something that is less verbose, in addition, this is a stateful variable so let's use `useState`